scripts/configure-httpd-ipa.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/bash
+
+IRONIC_CERT_FILE=/certs/ironic/tls.crt
```

Comment:
For consistency should we be able to override this?

```suggestion
IRONIC_CERT_FILE=${IRONIC_CERT_FILE:-/certs/ironic/tls.crt}
```

Reply:
Fixed! Thanks